dbt/adapters/databricks/impl.py

```diff
@@ -425,60 +431,54 @@ def parse_columns_from_information(  # type: ignore[override]
     def get_catalog(
         self, manifest: Manifest, selected_nodes: Optional[Set] = None
```

Comment:
Noting for my own sake that in the previous implementation of this method, `selected_nodes` was optional but was never enforced.

Reply:
I believe `selected_nodes` was added recently for the `get_catalog` performance improvements. We had a miss where we called it with the additional argument before checking if we could and @benc-db had to update this signature as a result. So the kwarg did nothing besides allowing `get_catalog` to be called with an additional argument.